rholang/src/main/scala/coop/rchain/rholang/interpreter/Reduce.scala

```diff
@@ -45,7 +47,8 @@ object Reduce {
   class DebruijnInterpreter[M[_]: InterpreterErrorsM, F[_]](
       tupleSpace: IStore[Channel, BindPattern, Seq[Channel], TaggedContinuation],
       dispatcher: => Dispatch[M, Seq[Channel], TaggedContinuation])(
-      implicit parallel: cats.Parallel[M, F])
+      implicit parallel: cats.Parallel[M, F],
+      captureM: Capture[M])
```

Comment:
If we don't need to call `captureM` explicitly can we move this constraint to the `[M[_] : _]` part of the signature? ?

Reply:
Fine with me.  I think it's a taste thing - I don't really like the context bounds syntax, and I tend to use the parameters directly more often than not.

If you want to do it that way here, I will do.

Reply:
Context bounds are just less typing so that's why go for it if I don't have to use instance of the type class. Thanks.